### PR TITLE
Directory crash fix and universal iso

### PR DESCRIPTION
**Pull request by Zedb0T**

### CODE REVIEW

File: resources/openGOALModLauncher.py [hunk 1]
```diff
@@ -105,11 +112,20 @@ def try_remove_dir(dir):
...
+		#if ISO_DATA is empty, prompt for their ISO and store its path.
+		root = tk.Tk()
+		print("Please select your iso.")
+		root.title("Select ISO")
+		root.geometry('230x1')
+		iso_path = filedialog.askopenfilename()
+		root.destroy()
+		if (pathlib.Path(iso_path).is_file):
+			if ((not pathlib.Path(iso_path).suffix == '.iso')):
+				1/0
```
Comment (dallmeyer) [inline resources/openGOALModLauncher.py:127]: nice

Comment (dallmeyer) [reply resources/openGOALModLauncher.py:127]: you could add your "path contains the word 'emulator'" check in here too :P

File: resources/openGOALModLauncher.py [hunk 2]
```diff
@@ -120,17 +136,20 @@ def try_remove_dir(dir):
...
+	
+	
+	#move the extrated contents to the universal launchers directory for next time.
+	if (not (exists(( UniversalIsoPath + "\jak1\Z6TAIL.DUP")))):
+		while (process_exists("extractor.exe")):
+			time.sleep(1)
+	if not (exists(( UniversalIsoPath))):
+		#os.makedirs(AppdataPATH + "\OpenGOAL-Launcher\data\iso_data")
+		print("The new directory is created!")
+		shutil.move(InstallDir + "/data/iso_data", "" + UniversalIsoPath +"")
```
Comment (dallmeyer) [inline resources/openGOALModLauncher.py:151]: what's with the extra quotations before/after `UniversalIsoPath`?

**Review by dallmeyer — APPROVED** [review]
LGTM, one minor thing

File: resources/openGOALModLauncher.py (hunk 2 above)
Comment (Zedb0T) [reply resources/openGOALModLauncher.py:151]: UNIX or whatever, incase the user has spaces in a directory name (I think?)

**Pull request merged**